### ISSUE TITLE
Coupon field is not discoverable by screen reader users before getting to Checkout button

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13590]
**Describe the bug**

- The coupon field is after the checkout button, therefore, it might go unnoticed by a blind user if the SR doesn't vocalize it, before hitting the "Proceed" button.
- This is information the user definitely needs to hear, check with designers and devs to see if we can place this better, basically anywhere before the "Proceed to Checkout" button.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Add product to cart.
3. Go to the cart page and **pretend you cannot see what is on the screen.**
4. Checkout products using the keyboard.
5. Listen how you miss the coupon field when reaching the **Proceed to Checkout** button.

**Expected behavior**

- Coupon field is discoverable by screen reader users before getting to the **Proceed to Checkout** button

**Screenshots**
![2020-11-24_16-30-38](https://user-images.githubusercontent.com/15113219/131574950-e7eaf8c4-1cc9-4925-a022-c720e5784cd9.jpg)

**Comment by saad-mo** [comment]
@developpeurweb should the couple component be moved before the checkout button? Mabe, it's a UX question as well

If yes, a new outlet needs to be created in `cx-cart-totals` after `cx-order-summary` summary

**Comment by developpeurweb** [comment]
@saad-mo Yes, IMO the coupon field is outside the user journey, especially while on keyboard navigation, therefore SRs.
I remember discussing this with @Manurod2018 quite a while ago, but please ping him so he is in the loop.
The latest mockups seem to have the coupon field well placed now. Before the "Proceed" button.

![Screen Shot 2021-10-13 at 1 42 21 PM](https://user-images.githubusercontent.com/15113219/137186004-5dbbd69a-81f1-4422-8cfa-3386d99094b7.png)
